addons/hr_homeworking/static/src/calendar/common/popover/calendar_common_popover.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { patch } from "@web/core/utils/patch";
+import { AttendeeCalendarCommonPopover } from "@calendar/views/attendee_calendar/common/attendee_calendar_common_popover";
+import { Field } from "@web/views/fields/field"
+patch(AttendeeCalendarCommonPopover.prototype, "hr_homeworking_calendar_common_popover",{
+
+    setup() {
+        this.fieldNames = ["work_location_id", "work_location_name", "work_location_type", "employee_id", "weekday", "weekly", "date"];
+        this._super(...arguments)
+    },
+    isWorkLocationEvent(){
```

Comment:
You can define it as a  [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get):
That way the `()`  are not needed in the template :-)

```suggestion
    get isWorkLocationEvent() {
```

Reply:
At the beginning, I tried to do that but I had a lot of problems :/ When I spoke with mcm from JS Framework he said to use a function and not a getter

Reply:
It's a hidden issue with the `patch` function when we add a new getter that didn't exist on the base class.
For the moment, it's better to keep the method version.